packages/menu-surface/index.tsx

```diff
@@ -217,15 +212,16 @@ class MenuSurface extends React.Component<MenuSurfaceProps, MenuSurfaceState> {
         this.previousFocus.focus();
       },
       isFirstElementFocused: () =>
-        this.firstFocusableElement &&
+        Boolean(this.firstFocusableElement) &&
```

Comment:
Does't typescript has sugar syntax for null checking? 

`this.firstFocusableElement? === document.activeElement`

Reply:
actually I don't even need the Boolean() part. `firstFocusableElement` and `last` will be an HTMLElement or null. Which means that in all cases `this.firstFocusableElement === document.activeElement` will be valid.